TROUBLESHOOTING.rst

```diff
@@ -147,4 +145,4 @@ can be resolved on Ubuntu by running:
 .. bash::
 
     # Please modify command to match the version of python you use for development.
-    sudo apt-get install python3.6-dev
+    sudo apt-get install python3.8-dev
```

Comment:
This change is unrelated, but I make it here in order to prevent conflicts with:
https://github.com/kensho-technologies/graphql-compiler/pull/712

Reply:
(This is exactly what Bojan said he would do in that PR).

Reply:
#712 seems like low hanging fruit - let's try to merge it in now or keep the fix here? cc @bojanserafimov

Reply:
Let's make this one supersede #712, they both do essentially the same thing and this PR is more comprehensive.